docs/manual/src/docs/asciidoc/_includes/servlet/authentication/x509.adoc

```diff
@@ -15,7 +15,7 @@ It maps the certificate to an application user and loads that user's set of gran
 
 You should be familiar with using certificates and setting up client authentication for your servlet container before attempting to use it with Spring Security.
 Most of the work is in creating and installing suitable certificates and keys.
-For example, if you're using Tomcat then read the instructions here https://tomcat.apache.org/tomcat-6.0-doc/ssl-howto.html[https://tomcat.apache.org/tomcat-6.0-doc/ssl-howto.html].
+For example, if you're using Tomcat then read the instructions here https://tomcat.apache.org/tomcat-10.0-doc/ssl-howto.html[https://tomcat.apache.org/tomcat-10.0-doc/ssl-howto.html].
```

Comment:
Since Spring Boot [requires a minimum of Tomcat 9](https://docs.spring.io/spring-boot/docs/2.3.3.RELEASE/reference/htmlsingle/#getting-started-system-requirements-servlet-containers), I think it's best to point to the Tomcat 9. Would you please update this accordingly?

Reply:
I understand your comments. I will change this point.
Thank you.

Reply:
I completed fix my pull request as your comment.
Thanks.